assets/large_language_models/rag/components/validate_deployments/spec.yaml

```diff
@@ -33,7 +33,7 @@ inputs:
     optional: true
   acs_config:
     type: string
-    optional: false
+    optional: true
```

Comment:
Is this really optional?

how can we handle it in the code?

https://msdata.visualstudio.com/Vienna/_git/AzureMlCli?path=/src/azureml-rag/src/azureml/rag/tasks/validate_deployments.py&version=GBtest1&line=657&lineEnd=658&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents